Inferred services private beta guide - Fix agent configuration instructions (#24012)
DataDog/documentation: content/en/tracing/guide/inferred-service-opt-in.md

Requirements:
- Datadog Agent version >= [7.50.3][4].

Update your `datadog.yaml` configuration file with the following:

{{< code-block lang="yaml" filename="datadog.yaml" collapsible="true" >}}

apm_config:
  compute_stats_by_span_kind: true
  peer_tags_aggregation: true
  peer_tags: ["_dd.base_service","amqp.destination","amqp.exchange","amqp.queue","aws.queue.name","aws.s3.bucket","bucketname","cassandra.keyspace","db.cassandra.contact.points","db.couchbase.seed.nodes","db.hostname","db.instance","db.name","db.namespace","db.system","grpc.host","hostname","http.host","http.server_name","messaging.destination","messaging.destination.name","messaging.kafka.bootstrap.servers","messaging.rabbitmq.exchange","messaging.system","mongodb.db","msmq.queue.path","net.peer.name","network.destination.name","peer.hostname","peer.service","queuename","rpc.service","rpc.system","server.address","streamname","tablename","topicname"]

{{< /code-block >}}

> You can also use the following environment variables in your Datadog Agent launch configuration:

Reviewer: ```suggestion
Alternatively, configure this by setting the following environment variables in your Datadog Agent launch configuration:
```
